tools/ci_build/github/linux/ort_minimal/build_check_binsize_config/android_minimal_basic_with_some_ops.config

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
these build options are somewhat arbitrary, open to other suggestions

Reply:
I'd be inclined to leave ABSL enabled given it provides better memory usage/perf doesn't it?